### ISSUE TITLE
Need down/down (space character) for semaphore decoder

### ISSUE THREAD

**Issue by amdbeck** [puzztool/puzztool#136]
Need down/down (space character) for semaphore decoder

**Comment by kfarnung** [comment]
Can you clarify a bit for the use case? You can insert a space by hitting next without a valid combination selected.

**Comment by amdbeck** [comment]
When entering this
[image: image.png]
in the semaphore click-on-the position, there is no way to enter the third,
fourth, or sixth character.  Would be nice if puzztool had a way to enter
that character and tell me that it is a space, since I don't have that
memorized.

**Comment by kfarnung** [comment]
I don’t know of a good way to do it at the moment other than to just have it default to the “rest” position if nothing is selected. It currently has no way to choose the same position twice.

**Comment by amdbeck** [comment]
Makes sense.  Husband just told me to go ahead and submit it as feedback,
so I did.

- AB

**Comment by kfarnung** [comment]
Totally fair, just wanted to understand the context. Rest/space doesn’t come up often.
